CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### New Features
 
+- new cli flag `-o`/`--os` to override auto-detected operating system for a sample @captainGeech42
```

Comment:
i think we should avoid using `-o` because thats commonly used by other tools for "output" or similar. `--os` should be sufficient and it only costs an extra two keypresses :-)

Reply:
```suggestion
- new cli flag `--os` to override auto-detected operating system for a sample @captainGeech42
```